design/tenant_updates.go

```diff
@@ -0,0 +1,61 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("tenant_updates", func() {
+	a.BasePath("/tenant/updates")
+
+	a.Action("show", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.GET(""),
+		)
+		a.Headers(func() {
+			a.Header("Authorization", d.String, "the authorization header")
```

Comment:
because I need to be able to specify the `Authorization` header in the test, and with this `Header` func, the generated method to test the endpoint will provide me with an arg to pass the value. See https://github.com/fabric8-services/admin-console/pull/48/files#diff-a2bd8cd90ce50fc5463d8b3539620c92R83

Reply:
You don't really have to specify the Authorization header explicitly. You can just embed the token into the context as we do everywhere else. Am I missing something?

Reply:
Yes, I know this is what we usually do with our endpoint tests, but in this particular case it did not work: when testing the proxying with gork, there was no `Authorization` header so Gork would not match the HTTP request sent to the remote cluster service.